File: share-code/bai-tap-lap-trinh/python/condition/50.py
your_number = 4
str_one = "dương"
str_two = "chẵn"

if your_number < 0:
    str_one = "âm"

if your_number % 2 != 0:
    str_two = "lẻ"

print(f"Số nguyên {str_one} {str_two}")

File: share-code/bai-tap-lap-trinh/cpp/50.cpp
#include <iostream>
using namespace std;
int main()
{
    // Số nguyên dương > 0 || nguyên âm < 0
    // chẵn % 2 == 0 || lẻ % 2 != 0
    int yourNumber      = 4;
    string strName      = " dương";
    string strEven      = " chẵn";

    if (yourNumber < 0)
    {
        strName = " âm";
    }
    
    if (yourNumber % 2 != 0 )
    {
        strEven = " lẻ";
    }
    
    cout << "Số nguyên" << strName << strEven;
    
}